lesson08/assignment/my-app-name/src/views/Accordion.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div id="accordion">
+    <div class="card" v-for="(card, index) in cards" v-bind:key="card">
+      <div class="card-header" :id="'heading'+ (index + 1)">
+        <h5 class="mb-0">
+          <button class="btn btn-link" data-toggle="collapse" :data-target="'#collapse'+(index + 1)" aria-expanded="false" :aria-controls="'collapse'+(index + 1)">
+            Accordion Item #{{ index + 1 }}
+          </button>
+        </h5>
+      </div>
+      <div :id="'collapse'+(index + 1)" class="collapse" :aria-labelledby="'heading'+ (index + 1)" data-parent="#accordion">
+        <div class="card-body">
+          <!--I dont know how safe this is but it helps me put in any sort of element I want as long as I explicity declare the tags
+          If there are problems with this down the line I can always just have the list items start and end with element tags -->
+          <span v-html="card"></span>
```

Comment:
Dynamically generated HTML from string data like this is something to be cautious with. The danger comes when it's generated from data that users input, as that opens the possibility of malicious code being executed. If it's just coming from hard data defined in the same file, it's safe.